.github/workflows/c.yaml

```diff
@@ -12,38 +12,34 @@ on:
       - "c/**"
       - .github/workflows/c.yaml
 jobs:
-
   ubuntu-lts:
-
     strategy:
       matrix:
-        build_type: [ "Debug", "Release" ]
-        compiler: [
-          { c: "gcc-6", cxx: "g++-6", package: "gcc-6 g++-6" },
-          { c: "clang-6.0", cxx: "clang++-6.0", package: "clang-6.0" }
-        ]
-        build_shared_libraries: [ true, false ]
-        exclude: [
```

Comment:
I digged in the history and this combination was excluded because at some point, there was a "code coverage" job that covered this combination. This job is not existing anymore, so I'm adding the combination here.